package.json

```diff
@@ -42,7 +42,7 @@
     "packages/*"
   ],
   "engines": {
-    "node": ">=18.14.1",
+    "node": ">=18.20.1",
```

Comment:
Should this also drop support for 19 (1.49% of users on 4.x in the last 30 days) and `20.0.0`, `20.1.0` and `20.2.0` (1% of users on 4.x in the last 30 days) so that we can finally upgrade Sharp?

Reply:
> Should this also drop support for 19 (1.49% of users on 4.x in the last 30 days) and 20.0.0, 20.1.0 and 20.2.0 (1% of users on 4.x in the last 30 days) so that we can finally upgrade Sharp?

I think dropping support for `19` is fine and in line with our Node.js support policy. I am not sure about `20`, but if we consider that we never set a restriction, I think we should be careful about this change. Personally, I would just set `>=20`

Reply:
20.3.0 added support for node-api v9, which is required for not only Sharp, but also other common native binaries (it was equivalent to 18.17.0, came out at the same time)